images/release/Dockerfile

```diff
@@ -15,13 +15,17 @@ ENV VERSION=1.6 \
 ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
 RUN mkdir $TMPDIR && \
-    INSTALL_PKGS="make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq openssl" && \
+    INSTALL_PKGS="make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito openssl" && \
```

Comment:
Not built anymore, we locked when centos dropped their Go support.  Will remove when we stop building 1.3